fem/coefficient.hpp

```diff
@@ -946,6 +946,73 @@ class OuterProductCoefficient : public MatrixCoefficient
                      const IntegrationPoint &ip);
 };
 
+class QuadratureFunction;
+
+/// Quadrature function vector coefficient
+class VectorQuadratureFunctionCoefficient : public VectorCoefficient
+{
+private:
+   QuadratureFunction *QuadF; //do not own
+   int index;
+   int length;
+
+public:
+   /// constructor with a quadrature function as input
+   VectorQuadratureFunctionCoefficient(QuadratureFunction *qf);
+
+   /// constructor with a null qf
+   VectorQuadratureFunctionCoefficient() : VectorCoefficient(0) { QuadF = NULL; }
```

Comment:
This constructor leaves index and length undefined.

Reply:
Should be fixed now